src/test/FuzzerImpl.cpp

```diff
@@ -1387,25 +1452,40 @@ TransactionFuzzer::initialize()
                     PublicKey issuer;
                     FuzzUtils::setShortKey(issuer, trustLine.mAssetID.mIssuer);
 
-                    // Reduce "account"'s balance by paying the issuer.
+                    // Reduce "account"'s balance of this asset by paying the
+                    // issuer.
                     auto tle = stellar::loadTrustLine(ltx, account, asset);
                     auto const availableTLBalance =
                         tle.getAvailableBalance(ltx.loadHeader());
                     auto const targetAvailableTLBalance =
                         trustLine.mAssetAvailableForTestActivity;
+                    auto const paymentAmount =
+                        availableTLBalance - targetAvailableTLBalance;
 
                     if (availableTLBalance > targetAvailableTLBalance)
                     {
-                        auto reduceNonNativeBalanceOp = txtest::payment(
-                            issuer, asset,
-                            availableTLBalance - targetAvailableTLBalance);
+                        auto reduceNonNativeBalanceOp =
```

Comment:
Agreed -- that's another place where there should be an assertion (instead of an `if`) to catch it if setup isn't doing what it intends to.  I've made that change.